packages/integration-tests/sfnStep.js

```diff
@@ -92,6 +92,31 @@ class SfnStep {
     return scheduleEvents.map((e) => this.getStepExecutionInstance(executionHistory, e));
   }
 
+  /**
+   * Gets the input to the step by looking for the 'schedule' for the given step
+   * and returning the parsed input object.
+   *
+   * @param   {String} workflowExecutionArn - AWS Execution ARN of the step function execution
+   * @param   {String} stepName             - Name of the workflow step of interest
+   * @returns {Object}                      - Parsed JSON string of input to step with <stepName>
+   *                                          from the workflow execution of interest.
+   */
+  async getStepInput(workflowExecutionArn, stepName) {
```

Comment:
@abarciauskas-bgse  What can I go look at/read to discover how the statefunctions execution steps progress.  I would never have known to go looking in the LambdaFunctionScheduled.  (although I think I should look at the Execution event hisory some).  Thanks

Reply:
Yup this page is hard to find but very helpful: https://docs.aws.amazon.com/step-functions/latest/apireference/API_Types.html